contributors/devel/sig-release/cherry-picks.md

```diff
@@ -61,7 +61,13 @@ If the change is in cloud provider-specific platform code (which is in the
 process of being moved out of core Kubernetes), describe the customer impact,
 how the issue escaped initial testing, remediation taken to prevent similar
 future escapes, and why the change cannot be carried in your downstream fork of
-the Kubernetes project branches.
+the Kubernetes project branches. The changes are only allowable when:
+
+- No new features are introduced (e.g. changing Kubernetes API or supporting new
+  Kubernetes features)
```

Comment:
@tpepper do you think "adding a new annotation" is adding a new feature? annotations are pretty useful to enable or disable things while won't break other scenarios.

Reply:
Adding a default annotation into k/k?  I'd like to think of cloud provider specific annotations as an operational pattern and something that wouldn't be explicitly in k/k.

Reply:
yea, I mean cloud provider specific annotations here.

Reply:
let me add this into list above

Reply:
Updated per above discussion. @tpepper could you have a look again?